source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/show/details/index.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+import { withData as withOrbit } from 'react-orbitjs';
+
+import {
+  ProjectResource, attributesFor, OrganizationResource,
+  UserResource, GroupResource
+} from '@data';
+import { i18nProps, withTranslations } from '@lib/i18n';
+
+import Products from './products';
+
+interface IExpectedProps {
+  project: ProjectResource;
+}
+
+interface IComposedProps {
+  group: GroupResource;
+  organization: OrganizationResource;
+  owner: UserResource;
+  organizationOwner: UserResource;
+}
+
+type IProps =
+  & IExpectedProps
+  & IComposedProps
+  & i18nProps;
+
+class Details extends React.PureComponent<IProps> {
+  render() {
+    const { t, organization, owner, project, organizationOwner } = this.props;
+    const { description } = attributesFor(project);
+    const { name: organizationName, logoUrl } = attributesFor(organization);
+    const { name: orgOwnerName } = attributesFor(organizationOwner);
+
+    const logo = logoUrl
+      ? <img src={logoUrl} style={{ maxHeight: '60px', maxWidth: '60px' }} />
+      : '\u00A0';
+
+    return (
+      <div className='flex-grow p-r-lg-lg'>
+        <div data-test-public-project-details className='m-b-lg'>
+
+          <div className='m-b-md thin-bottom-border p-b-lg flex-row justify-content-start align-items-center'>
+            <span data-test-logo>{logo}</span>
+
+            <div className='p-l-lg'>
+              <h2 data-test-org-name className='p-r-sm p-t-sm p-b-none'>
+                {organizationName}
+              </h2>
+              <div data-test-org-owner className='italic'>
+                <label>{t('project.organizationContact')}</label>
+                <strong>{orgOwnerName}</strong>
+              </div>
+
+              <div data-test-project-owner className='italic'>
+                <label>{t('project.projectOwner')}</label>
+                <strong>{attributesFor(owner).name}</strong>
+              </div>
+            </div>
+          </div>
+
+          <div data-test-description className='p-b-lg fs-16'>
+            {description}
+          </div>
+        </div>
+
+
+        <Products project={project} />
+      </div>
+    );
+  }
+}
+
+export default compose<IProps, IExpectedProps>(
+  withTranslations,
+  withOrbit((passedProps: IExpectedProps) => {
+    const { project } = passedProps;
```

Comment:
can be refactor into a shorter version :)

Reply:
:-\